src/components/sections/items/ActionSectionItem.tsx

```diff
@@ -12,12 +12,10 @@ import {ThemeIcon} from '@atb/components/theme-icon';
 import {useSectionItem} from '../use-section-item';
 import {SectionItemProps} from '../types';
 import {useSectionStyle} from '../use-section-style';
-import {InternalLabeledSectionItem} from './InternalLabeledSectionItem';
-import {FixedSwitch} from '@atb/components/switch';
 import {InteractiveColor} from '@atb/theme/colors';
 import {SvgProps} from 'react-native-svg';
 
-type ActionModes = 'check' | 'toggle';
+type ActionModes = 'check';
```

Comment:
Do we need the `ActionModes` prop when there is only one option? Is it just in case we add more in the future?